<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>styledMessageBox</class>
 <widget class="QDialog" name="styledMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/****************
 *         v3.0            *
 ****************/

QWidget
{
   font:  12pt &quot;DINPro-Medium&quot;;
}

QWidget#backgroundWidget
{
    background: rgb( 128, 128, 128 );
}

QLabel#titlebarLabel {
	background: rgb( 51, 51, 51 );
}
QLabel#titlebarTextLabel {
    font: 24pt &quot;DINPro-Medium&quot;;
    color: white;
}

QLabel#instructionsLabel
{
    background: rgb( 191, 191, 191 );
}

QLabel#instructionsTextLabel
{
    font: 12pt &quot;DINPro-Medium&quot;;
    color: black;
}

/* QPushButton */

QPushButton {
    background: rgb( 88, 89, 91 );
    border: 1px solid rgb( 88, 89, 91 );
    color: white;
    font: 12pt &quot;DinPRO-Medium&quot;;
    min-width: 164px;
    max-width: 164px;
    min-height: 42px;
    margin: 1px;
    text-align: center;
    border-radius: 10px;
}

QPushButton::hover 
{
    background: rgb(52, 73, 94 );
    border-color: rgb( 166, 166, 166 );
    color: rgb( 214, 214, 214);
    border-width: 1px;
}

 /* make the default button prominent */
 QPushButton:default {
     background:  rgb( 52, 73, 94 );
     border: 1px solid   rgb( 52, 73, 94 );
	 font: 17px &quot;DinPro-bold&quot;;
 }


QPushButton:disabled 
{
    background:  rgb( 51, 51, 51 ); 
    border: rgb( 51, 51, 51 ); 
 }

QDialogButtonBox
{
   button-layout: 1;
}</string>
  </property>
  <widget class="QWidget" name="backgroundWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="titlebarTextLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>STYLIZED MESSAGE TITLE</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructionsTextLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>STYLIZED MESSAGE INSTUCTIONS</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructionsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>581</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titlebarLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>342</width>
      <height>46</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="messageText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>msg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>instructionsLabel</zorder>
   <zorder>titlebarLabel</zorder>
   <zorder>titlebarTextLabel</zorder>
   <zorder>instructionsTextLabel</zorder>
   <zorder>buttonBox</zorder>
   <zorder>messageText</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>styledMessageBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>styledMessageBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
